src/pages/teachers/programmes/[programmeSlug]/units.tsx

```diff
@@ -45,11 +47,19 @@ import { toSentenceCase } from "@/node-lib/curriculum-api-2023/helpers";
 import NewContentBanner from "@/components/TeacherComponents/NewContentBanner/NewContentBanner";
 import PaginationHead from "@/components/SharedComponents/Pagination/PaginationHead";
 import { TierSchema } from "@/node-lib/curriculum-api-2023/queries/unitListing/tiers/tiers.schema";
+import SubjectCategoryFilters from "@/components/TeacherComponents/SubjectCategoryFilters";
+import YearGroupFilters from "@/components/TeacherComponents/YearGroupFilters";
 
 export type UnitListingPageProps = {
   curriculumData: UnitListingData;
 };
 
+const StyledFieldset = styled.fieldset`
+  border: 0px;
+  margin: 0;
+  padding: 0;
+`;
```

Comment:
think we've used this in a few places now probably worth adding to components lib (tech debt, doesn't need to be now)

Reply:
agreed 👍